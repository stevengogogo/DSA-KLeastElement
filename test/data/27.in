10 50000
8090 -1133 4697 7527 283 1271 9954 6664 1159 -8276
Reverse 9 10
Insert 6 -1405
Query 9 9 1
Query 11 11 1
Insert 5 -9405
Delete 5
Delete 6
Delete 7
Delete 2
Reverse 4 6
Delete 4
Delete 3
Reverse 1 3
Query 4 6 2
Delete 6
Reverse 1 3
Delete 3
Query 3 4 2
Query 1 1 1
Reverse 1 1
Delete 1
Reverse 2 2
Reverse 2 3
Delete 3
Insert 1 8805
Reverse 3 3
Reverse 1 1
Insert 1 -4079
Query 4 4 1
Query 4 4 1
Insert 1 8776
Delete 4
Delete 2
Query 2 2 1
Delete 2
Delete 2
Reverse 1 1
Delete 1
