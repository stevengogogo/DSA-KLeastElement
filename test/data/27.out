6664
1159
283
283
8090
-8276
-8276
8805
